modules/cluster-logging-collector-log-forward-syslog.adoc

```diff
@@ -95,6 +95,48 @@ spec:
 $ oc create -f <file-name>.yaml
 ----
 
+[id=cluster-logging-collector-log-forward-examples-syslog-log-source]
+== Adding log source information to message output
+
+You can add namespace_name, pod_name, and container_name to the `message` field of the record by adding the `AddLogSource` field to your `ClusterLogForwarder` CR.
+
```

Comment:
.Procedure

* Add `AddLogSource: true` to your `ClusterLogForwarder` CR.

Reply:
@libander this seems like an assembly that should have three modules: 1. A procedure module for 'Configuring log forwarding using the syslog protocol' or something on those lines. 2. A procedure module with Rolfe's suggestion above for this section - 'Adding log source information to message output' (procedure module since it starts with a gerund.) and 3. A reference module for Syslog parameters.
Currenlty, this whole section: Forwarding logs using the syslog protocol is marked as content-type: PROCEDURE, which is misleading. 
Considering the parent section, I believe, you might need to use nested assemblies under the assembly for the parent section: https://deploy-preview-42331--osdocs.netlify.app/openshift-enterprise/latest/logging/cluster-logging-external.html
While this may be outside the scope of this PR I suggest you file a follow up enhancement Jira issue for this.
However for this Jira, I suggest you implement this recommendation by @rolfedh